pkg/connector/connector.go

```diff
@@ -39,6 +39,16 @@ func (c *Postgresql) Metadata(ctx context.Context) (*v2.ConnectorMetadata, error
 	return &v2.ConnectorMetadata{
 		DisplayName: "Postgresql",
 		Annotations: annos,
+		AccountCreationSchema: &v2.ConnectorAccountCreationSchema{
+			FieldMap: map[string]*v2.ConnectorAccountCreationSchema_Field{
+				"email": {
+					DisplayName: "email",
+					Required:    true,
+					Description: "email address",
```

Comment:
This should be sentence-cased, we're going to show this to users of this schema. Something like:

```This email will be used as the login for the new user.```